region/maridia/outer/Fish Tank.json

```diff
@@ -558,6 +643,30 @@
       ],
       "devNote": "With a longer runway there are fewer tech requirements to the jump, but at this difficulty they shouldn't matter."
     },
+    {
+      "link": [2, 1],
+      "name": "Come in With Water Shinecharge, Leave With Temporary Blue",
+      "entranceCondition": {
+        "comeInRunning": {
+          "speedBooster": true,
+          "minTiles": 0.4375
+        }
+      },
+      "requires": [
+        "canWaterShineCharge",
+        "canStationaryLateralMidAirMorph",
+        {"or": [
+          "HiJump",
+          "canTrickySpringBallJump"
```

Comment:
Yeah, both HiJump-only and spring-ball-only are pretty terrible. It looks like in both cases the tightest part is the left-to-right jump across the gap just above the bottom Pirate. That jump seems to be tighter with HiJump than spring ball, but with spring ball it takes more time per attempt because of all the pausing. Added "canBeVeryPatient" for both versions to bump them to insane, leaving just the HiJump+SpringBall version as the extreme version